.github/workflows/smoketests.yml

```diff
@@ -28,7 +28,7 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           
     - name: Checkout external repository with Cypress tests
-      uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11
+      uses: actions/checkout@v4
```

Comment:
Can you revert this back please, we need to pin the action versions for security.

```suggestion
      uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11
```

Reply:
This doesn't seem to work across orgs though unfortunately. This is current how the binary-com/deriv-com yml is currently working: 
![image](https://github.com/binary-com/deriv-app/assets/144238191/ef2f8962-dc03-40e3-af49-01a6e174f7eb)